Confirmation before sending image via Cmd/Ctrl+V

veniversum opened the pull request:
Fixes #196.

sindresorhus commented:
Can you add this to the readme too?

sindresorhus commented on index.js:
```diff
@@ -153,6 +154,24 @@ app.on('ready', () => {
+			electron.dialog.showMessageBox({
+				type: 'info',
+				buttons: ['Send', 'Cancel'],
+				message: 'Are you sure you want to send image in clipboard?',
```
> Are you sure you want to send the image in the clipboard?

veniversum commented:
Rebased and added edits as requested. If you want to include a screenshot in the readme, I think it would be better if you do it since that'll maintain the look and feel from the other screenshots.

sindresorhus commented:
I don't think we need a screenshot for this.

sindresorhus commented:
Thanks :)

You done some good pull requests now. Would you be interested in joining the project? No worries if not.

veniversum commented:
Yes I would!

sindresorhus commented:
Welcome! I'm on Gitter if you want to discuss anything in realtime. https://gitter.im/sindresorhus/meta
